utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+from os import path
+
+
+def save_answer_on_json_file(answer, json_file_name):
+    """
+    Save the answer in a JSON file.
+
+    Args:
+        answer (str): The answer to be saved.
+        json_file_name (str): The name of the JSON file.
+    """
+    json.dump(answer, open(json_file_name, "w"))

```

Comment:
its bad practice opening a file like this.
make sure your file is closed using "with" or open and then close.